### ISSUE TITLE
Context for mark mode ?

### ISSUE THREAD

**Issue by aviatesk** [tuttieee/vscode-emacs-mcx#56]
Could you provide context for mark mode which is available in `when` clause ?

For example, say, in a case when we want to do define per-paragraph jump with [Space Block Jumpter](https://marketplace.visualstudio.com/items?itemName=jmfirth.vsc-space-block-jumper) package, we need [context](https://code.visualstudio.com/docs/getstarted/keybindings#_when-clause-contexts) like `inMarkMode` below available in `when` clause.

> keybindings.json
```json
/* Enable per-paragraph jump */
{
    "key": "ctrl+b",
    "command": "spaceBlockJumper.moveDown"
},
{
    "key": "ctrl+b",
    "command": "spaceBlockJumper.selectDown",
    "when": "inMarkMode && editorTextFocus"
}
```

This can be done to some extent with default `when: "editorHasSelection"` context, but it just works only when we have selection, not at the very start of mark mode...

**Comment by tuttieee** [comment]
Thank you for the suggestion. I will consider it.

memo:
* https://github.com/Microsoft/vscode/issues/10471
* https://github.com/VSCodeVim/Vim/blob/529dc2ddfbe03e96b61b15d7ecb2ec0c466c1a9b/src/util/vscode-context.ts

**Comment by tuttieee** [comment]
Released in v0.4.0!

**Comment by gpaul** [comment]
I came here looking exactly for this, thanks!
